1.21.6 & 1.21.7 Support

EspartaProJunior opened the pull request.

EspartaProJunior committed:
Updated CommandApi to 10.1.1

Wuason6x9 commented:
That's wrong, to change the version of a dependency you must also do it in the Dependencies class. A Library

Wuason6x9 commented:
That's wrong, to change the version of a dependency you must also do it in the Dependencies class. A Library

EspartaProJunior committed:
Update build.gradle.kts

EspartaProJunior commented:
where is Dependencies class?

EspartaProJunior committed:
Added CraftEngine API

EspartaProJunior committed:
Update Dependencies.java

EspartaProJunior committed:
Updated version

Wuason6x9 reviewed (changes requested):
It seems that everything is fine to update CommandAPI, but what is the reason why you want to update the API? Also, why do you add the craftengine API in mechanics? What is its utility/function here?

EspartaProJunior commented:
To be used in 1.21.6 and 1.21.7,

I checked the adapter and did not find anything related to compile it, if it already exists in the adapter, I remove it

Wuason6x9 commented:
![image](https://github.com/user-attachments/assets/b1ea55ff-9323-42af-946a-5ade0c713c2d)
Also, you need to add the dependency here. Located in the Mechanics.java file in the main package.

EspartaProJunior committed:
removed craftengine

EspartaProJunior commented:
It seems that only the adapter needs to be updated, I will open pull request there.

EspartaProJunior commented:
https://github.com/Wuason6x9/Adapter/pull/1

EspartaProJunior committed:
Added 1.21.6, 1.21.7 Support

EspartaProJunior committed:
Update build.gradle.kts

EspartaProJunior committed:
1.21.6-7

EspartaProJunior committed:
1.21.7 Support

I still can't get it to start because of CommandAPI errors.

EspartaProJunior committed:
fixes build.gradle

EspartaProJunior committed:
w

EspartaProJunior committed:
fixed nms

EspartaProJunior commented:
It works in 1.21.7 now

![image](https://github.com/user-attachments/assets/a40544aa-2293-4615-a991-ff78ffb39c12)

EspartaProJunior committed:
Added Support to 1.21.8

CommandAPI Still not support 1.21.8

EspartaProJunior committed:
Updated to CommandAPI 10.1.2

EspartaProJunior committed:
Updated Adapter to 1.0.6

Wuason6x9 committed:
Update library dependencies and Minecraft version mappings

Upgraded `item-nbt-api` to 2.15.1 and `Adapter` to 1.0.6.1 in both `Dependencies.java` and `build.gradle.kts`. Adjusted Minecraft version mappings for versions 1.21.6 to 1.21.8 to include correct data IDs.
